jcloud/parsers/remove.py

```diff
@@ -5,8 +5,17 @@ def set_remove_parser(parser=None):
     if not parser:
         parser = set_base_parser()
 
-    parser.add_argument(
-        'flows',
+    remove_group = parser.add_mutually_exclusive_group()
+
+    remove_group.add_argument(
+        '--phase',
+        help='The phase to filter flows on for removal',
+        type=str,
+        choices=['Pending', 'Starting', 'Updating', 'Serving', 'Paused', 'Failed'],
+    )
+
+    remove_group.add_argument(
+        '--flows',
```

Comment:
Please make this a positional arg like before. Else this would be a breaking change, which is rather unnecessary right now.

Reply:
This has also become a positional argument.